packages/core/upload/admin/src/components/AssetCard/tests/AssetCardBase.test.js

```diff
@@ -3,12 +3,10 @@ import { ThemeProvider, lightTheme } from '@strapi/design-system';
 import { render as renderRTL, fireEvent, waitFor } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 
-import en from '../../../translations/en.json';
-
 import { AssetCardBase } from '../AssetCardBase';
 
 jest.mock('react-intl', () => ({
-  useIntl: () => ({ formatMessage: jest.fn(({ id }) => en[id]) }),
+  useIntl: () => ({ formatMessage: jest.fn(({ defaultMessage }) => defaultMessage) }),
 }));
```

Comment:
I think we can remove this mock entirely. For new tests we already do this and rely on `defaultMessage` always.

Reply:
do you have an example of it?

Reply:
No, but if you remove the whole `jest.mock()` call, react-intl will render the `defaultMessage` always ...

Reply:
unfortunately it doesn't work with the removal of the ` jest.mock()` call

Reply:
It is wrapped in the intlProvider? It might not be the case this is why we are mocking it IMO

Reply:
Yes, the provider is missing. You need to wrap the component into:

```js
import { IntlProvider } from 'react-intl'; 

<IntlProvider locale="en" messages={messageForPlugin} defaultLocale="en">
  <ThemeProvider theme={lightTheme}>
        <AssetCardBase name="Card" extension="png" {...props} />
  </ThemeProvider>
</IntlProvider>
```

Reply:
> It is wrapped in the intlProvider? It might not be the case this is why we are mocking it IMO

Good to see you @soupette <3 Miss you

Reply:
done thanks @gu-stav ...nice to see you @soupette